docs/shared/features/enhance-tooling.md

```diff
@@ -0,0 +1,102 @@
+# Enhance Tooling
+
+Nx is able to enhance your existing tooling in a non-intrusive way so that, without modifying your tooling set-up, Nx can add:
+
+- [Fully configured caching](/features/cache-task-results)
+- [Task pipelines](/concepts/task-pipeline-configuration)
+- [Task distribution](/ci/features/distribute-task-execution)
+
+This works for any tool for which there exists an [official plugin](/plugin-registry).
+
+## Setup
+
+The following command will make Nx detect your tooling and set up the appropriate plugins:
+
+```shell
+nx init
```

Comment:
now this is mostly for migrating to Nx on existing repos. Which for an "enhance tooling" page might make sense. I wonder though whether we should also mention the CNW option

Reply:
`nx init` now will detect if there are packages that you are using for which we have official plugins and give you the option to install them.  You can run it on an existing Nx repo to detect plugins that you don't have installed that would be useful.

I think getting people in the habit of running `nx init` whenever they want to add something from Nx is really good DX.